Sources/MapboxMaps/Foundation/BaseMapView.swift

```diff
@@ -7,10 +7,18 @@ import Turf
 
 internal typealias PendingAnimationCompletion = (completion: AnimationCompletion, animatingPosition: UIViewAnimatingPosition)
 
+// swiftlint:disable:next type_body_length
```

Comment:
This will be removed in a follow-on PR that migrates more functionality from `BaseMapView` into `MapboxMap`